test/XRP/issued-currency-client-integration.test.ts

```diff
@@ -128,4 +129,51 @@ describe('IssuedCurrencyClient Integration Tests', function (): void {
       AccountRootFlag.checkFlag(AccountRootFlag.LSF_REQUIRE_AUTH, flags!),
     )
   })
+
+  it('createTrustline - adding a trustline with 0 value', async function (): Promise<
+    void
+  > {
+    this.timeout(timeoutMs)
+    const issuer = await XRPTestUtils.randomWalletFromFaucet()
+    // GIVEN an existing testnet account and an issuer's wallet
+    // WHEN a trustline is created with the issuer with a value of 0
+    await issuedCurrencyClient.createTrustline(
+      issuer.getAddress(),
+      'USD',
+      '0',
+      wallet,
+    )
+
+    const trustLines = await issuedCurrencyClient.getTrustLines(
+      wallet.getAddress(),
+    )
+
+    // THEN no trustlines are created.
+    assert.isArray(trustLines)
+    assert.isEmpty(trustLines)
+  })
+
+  it('createTrustline - adding a trustline with non-zero value', async function (): Promise<
+    void
+  > {
+    this.timeout(timeoutMs)
+    const issuer = await XRPTestUtils.randomWalletFromFaucet()
+
+    // GIVEN an existing testnet account and an issuer's wallet
+    // WHEN a trustline is created with the issuer with a positive value
+    await issuedCurrencyClient.createTrustline(
+      issuer.getAddress(),
+      'USD',
+      '1',
+      wallet,
+    )
+
+    const trustLines = await issuedCurrencyClient.getTrustLines(
+      wallet.getAddress(),
+    )
+
+    // THEN a trustline is created between the wallet and the issuer.
+    assert.isArray(trustLines)
+    assert.isNotEmpty(trustLines)
```

Comment:
can we attempt to scrutinize the details of this trustline and make sure they're what we expected?  Past integration tests have relied on vaguer success states because we were querying existing testnet addresses that we didn't necessarily have perfect control over... but in the case of perfect control like this i think we can make it tighter

Reply:
sure thing, that seems reasonable.